department-of-veterans-affairs/caseflow
# Fix hearing AOD

Resolves #7525

### Description
Three changes in this PR
1) We updated the AOD cases query to actually be a `where` on AOD cases.
2) We started passing whether a case was AOD to the frontend
3) We started displaying whether a case was AOD on the frontend

### Testing Plan
1. Go to the central office location
1. See that on both tabs we see some AOD cases.
